TombEngine/Game/savegame.cpp

```diff
@@ -113,7 +118,7 @@ Save::Vector3 FromVector3(const Vector3& vec)
 	return Save::Vector3(vec.x, vec.y, vec.z);
 }
 
-Save::Vector3 FromVector3(const Vector3i& vec)
+Save::Vector3 FromVector3i(const Vector3i& vec)
```

Comment:
I don't fully understand why you have converted this one from `Vector3` to `Vector3i` while all other changes seemingly do the opposite (removing `i` everywhere). Can you elaborate?

Reply:
Previously, `FromVector3()` had two overloads, but this made it unclear which type was being converted. Instead, now there are two versions: `FromVector3()` and `FromVector3i()`.